### ISSUE TITLE
Unrecognized event: MESSAGE_REACTION_REMOVE_EMOJI

### ISSUE THREAD

**Issue by Menkalian** [DV8FromTheWorld/JDA#1852]
[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
[stack overflow]: https://stackoverflow.com/questions/tagged/java

## General Troubleshooting

  
- [x] I have checked for similar issues.
- [x] I have updated to the [latest JDA version][download].
- [x] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

## Bug Report

After calling `Message#clearReactions(String unicode)` with the emote encoded as Unicode characters, I receive this error message:
```
2021-09-30 22:58:59.691 DEBUG 7756 --- [inWS-ReadThread] n.d.j.internal.requests.WebSocketClient  : Unrecognized event:
{"op":0,"s":40,"t":"MESSAGE_REACTION_REMOVE_EMOJI","d":{"emoji":{"name":"⬇️","id":null},"guild_id":"254236409706053632","message_id":"893240468891119636","channel_id":"259021952767754250"}}
```

Since the event `MESSAGE_REACTION_REMOVE_EMOTE` is registered [here](https://github.com/DV8FromTheWorld/JDA/blob/ae17ecee16782c1f336badbc1d400ff55464ca60/src/main/java/net/dv8tion/jda/internal/requests/WebSocketClient.java#L1349), I'd assume Discord is inconsistent with it's event naming and `MESSAGE_REACTION_REMOVE_EMOJI` should be handled just like `MESSAGE_REACTION_REMOVE_EMOTE`. Therefore both events should be registered for processing by the same handler implementation (or this is considered a bug on discord's side).

### Expected Behavior

No error should occur and the eventhandlers should be called accordingly.

### Code Example or Reproduction Steps

```kotlin
fun main() {
    val jda = JDABuilder
        .createDefault(token)
        .setActivity(Activity.of(Activity.ActivityType.DEFAULT, "..."))
        .setAudioPool(Executors.newScheduledThreadPool(4))
        .setAutoReconnect(true)
        .setEnableShutdownHook(true)
        .setStatus(OnlineStatus.ONLINE)
        .addEventListeners(listener)
        .build()

    jda.awaitReady()
    val slightSmileEmoji = "\uD83D\uDE42"
    val msg = jda
        .getGuildById("") // Any guild
        ?.getTextChannelById("") // Any text Channel
        ?.sendMessage("example")
        ?.complete()
    msg?.clearReactions(slightSmileEmoji)?.complete()
}

val listener = object : ListenerAdapter() {
    override fun onMessageReactionRemove(event: MessageReactionRemoveEvent) {
        LoggerFactory.getLogger("test").info("removed")
    }
}
```
For reproducing the method `Message#clearReactions` has to be called (just removing your own reaction does not work).
The listener is in this example is never called.

### Code for JDABuilder or DefaultShardManagerBuilder Used

```kotlin
JDABuilder
            .createDefault(token)
            .setActivity(Activity.of(Activity.ActivityType.DEFAULT, "..."))
            .setAudioPool(Executors.newScheduledThreadPool(4))
            .setAutoReconnect(true)
            .setEnableShutdownHook(true)
            .setStatus(OnlineStatus.ONLINE)
            .addEventListeners(*listeners.toTypedArray())
            .build()
```

### Exception or Error

```
2021-09-30 22:58:59.691 DEBUG 7756 --- [inWS-ReadThread] n.d.j.internal.requests.WebSocketClient  : Unrecognized event:
{"op":0,"s":40,"t":"MESSAGE_REACTION_REMOVE_EMOJI","d":{"emoji":{"name":"⬇️","id":null},"guild_id":"254236409706053632","message_id":"893240468891119636","channel_id":"259021952767754250"}}
```

**Comment by DV8FromTheWorld** [comment]
So this is actually a typo/bug in JDA.

[This](https://github.com/DV8FromTheWorld/JDA/blob/development/src/main/java/net/dv8tion/jda/internal/requests/WebSocketClient.java#L1349) should be `_EMOJI`. `_EMOTE` is not an actual event.

[The rest of the code in JDA knows this](https://github.com/DV8FromTheWorld/JDA/blob/ae17ecee16782c1f336badbc1d400ff55464ca60/src/main/java/net/dv8tion/jda/internal/handle/MessageReactionClearEmoteHandler.java#L45), it just happens to be that we typo'd this. Kinda crazy that no one has noticed this almost 3 years.

Feel free to open a PR against `development` please!

**Comment by DV8FromTheWorld** [comment]
Addressed by #1853

**Comment by Menkalian** [comment]
Thank you! The event is now processed correctly, but another error occurs:

```
2021-10-03 03:40:07.503 ERROR 11164 --- [inWS-ReadThread] n.d.j.internal.requests.WebSocketClient  : Got an unexpected error. Please redirect the following message to the devs:
	JDA 4.3.0_332
	MESSAGE_REACTION_REMOVE_EMOJI -> {"emoji":{"name":"⬇️","id":null},"guild_id":"317670715652308995","message_id":"894036059753680986","channel_id":"317670715652308995"}

java.lang.NumberFormatException: String value 894036059753680986 exceeds range of unsigned int.
	at java.base/java.lang.Integer.parseUnsignedInt(Integer.java:839) ~[na:na]
	at java.base/java.lang.Integer.parseUnsignedInt(Integer.java:928) ~[na:na]
	at net.dv8tion.jda.api.utils.data.DataObject.get(DataObject.java:830) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.api.utils.data.DataObject.getUnsignedInt(DataObject.java:586) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.internal.handle.MessageReactionClearEmoteHandler.handleInternally(MessageReactionClearEmoteHandler.java:59) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:36) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:952) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:839) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:817) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:991) ~[JDA-4.3.0_332.jar:4.3.0_332]
	at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385) ~[nv-websocket-client-2.14.jar:na]
	at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276) ~[nv-websocket-client-2.14.jar:na]
	at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996) ~[nv-websocket-client-2.14.jar:na]
	at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755) ~[nv-websocket-client-2.14.jar:na]
	at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108) ~[nv-websocket-client-2.14.jar:na]
	at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64) ~[nv-websocket-client-2.14.jar:na]
	at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45) ~[nv-websocket-client-2.14.jar:na]
```

I presume this is a typo [here](https://github.com/DV8FromTheWorld/JDA/blob/05a26f93dd3b749aef9aa6cb291825006d90871e/src/main/java/net/dv8tion/jda/internal/handle/MessageReactionClearEmoteHandler.java#L59).
 I can commit a bugfix myself, but that would only happen tomorrow (since it's already quite late for me).

**Comment by DManstrator** [comment]
Yeah, should be `content.getUnsignedLong("message_id")` instead.

**Comment by DV8FromTheWorld** [comment]
I agree with @DManstrator. Try making a PR for it :)

**Comment by Menkalian** [comment]
Issue fixed with version `4.3.0_333`
